api/src/main/java/org/pmiops/workbench/db/dao/NewUserSatisfactionSurveyDao.java

```diff
@@ -1,11 +1,7 @@
 package org.pmiops.workbench.db.dao;
 
-import java.util.Optional;
 import org.pmiops.workbench.db.model.DbNewUserSatisfactionSurvey;
-import org.pmiops.workbench.db.model.DbUser;
 import org.springframework.data.repository.CrudRepository;
 
 public interface NewUserSatisfactionSurveyDao
-    extends CrudRepository<DbNewUserSatisfactionSurvey, Long> {
-  Optional<DbNewUserSatisfactionSurvey> findByUser(DbUser user);
```

Comment:
was this removed because it's provided by default?

Reply:
no... looks like it is no longer used.  ok.

Reply:
I added this in the last PR thinking I'd need it for the eligibility endpoint. It's an artifact of splitting the story over multiple tickets instead of doing them all at once.